doc/manual/src/release-notes/rl-next.md

```diff
@@ -1,5 +1,7 @@
 # Release X.Y (202?-??-??)
 
+* Instead of "antiquotation", the more common term [string interpolation](../language/string-interpolation.md) is now used consistently.
+  Historical release notes were not changed.
```

Comment:
```suggestion
```

Reply:
Why remove that? Do you think it's self-evident? I added the note because the term _antiquotation_ still appears in the release notes, but one could have changed that as well.